### ISSUE TITLE
stdin: not in gzip format

### ISSUE THREAD

**Issue by slav123** [plopcas/hugo-s3-action#9]
```

Run plopcas/hugo-s3-action@v1.3.0
  env:
    AWS_REGION: ***
    AWS_ACCESS_KEY_ID: ***
    AWS_SECRET_ACCESS_KEY: ***
    MINIFY: false
/usr/bin/docker run --name a5c76b2a019d81a872446ea36f8527f9ea684a_e61005 --label a5c76b --workdir /github/workspace --rm -e "AWS_REGION" -e "AWS_ACCESS_KEY_ID" -e "AWS_SECRET_ACCESS_KEY" -e "MINIFY" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e...
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
gzip: stdin: not in gzip format
tar: Child returned status 1
```

**Comment by mehalick** [comment]
+1

Trying this for the first time and getting the same error.

https://github.com/mehalick/andy.mehalick.com/blob/main/.github/workflows/main.yml

```
Run plopcas/hugo-s3-action@v1.3.0
  env:
    AWS_REGION: us-east-1
    AWS_ACCESS_KEY_ID: ***
    AWS_SECRET_ACCESS_KEY: ***
    MINIFY: true
/usr/bin/docker run --name c044[2](https://github.com/mehalick/andy.mehalick.com/actions/runs/4692069510/jobs/8317324674#step:4:2)5b662f4a77c147[3](https://github.com/mehalick/andy.mehalick.com/actions/runs/4692069510/jobs/8317324674#step:4:3)aa17f1b[4](https://github.com/mehalick/andy.mehalick.com/actions/runs/4692069510/jobs/8317324674#step:4:4)eaa7fdac[5](https://github.com/mehalick/andy.mehalick.com/actions/runs/4692069510/jobs/8317324674#step:4:5)_f3d19c --label [6](https://github.com/mehalick/andy.mehalick.com/actions/runs/4692069510/jobs/8317324674#step:4:6)c0442 --workdir /github/workspace --rm -e "AWS_REGION" -e "AWS_ACCESS_KEY_ID" -e "AWS_SECRET_ACCESS_KEY" -e "MINIFY" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/andy.mehalick.com/andy.mehalick.com":"/github/workspace" 6c0442:5b662f4a[7](https://github.com/mehalick/andy.mehalick.com/actions/runs/4692069510/jobs/8317324674#step:4:8)7c1473aa17f1b4eaa7fdac5

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

**Comment by mikerinen** [comment]
+1
I came across this as well, but ended up changing to a different more maintained Github Action from the Marketplace.

It's probably the base image that should be updated.
AWS maintains a base image that could be updated
https://hub.docker.com/r/amazon/aws-cli

**Comment by plopcas** [comment]
This should be fixed on version v1.4.1. Thanks for your patience.
